book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -17,11 +17,10 @@ In `src` there's a `main.rs` file. Delete everything, then add these imports:
 ```rust,ignore
 extern crate amethyst;
 
-use amethyst::prelude::*;
 use amethyst::input::{is_close_requested, is_key_down};
-use amethyst::renderer::{DisplayConfig, DrawFlat, Event, KeyboardInput,
-                         Pipeline, PosTex, RenderBundle, Stage,
-                         VirtualKeyCode, WindowEvent};
+use amethyst::prelude::*;
+use amethyst::renderer::{DisplayConfig, DrawFlat, Event, Pipeline,
```

Comment:
Did you make sure the imports you removed are not used at all during the remaining of the tutorial?

Reply:
I haven't worked my way through the entire tutorial yet, but these imports (`WindowEvent`, `KeyboardInput`) make sense to me to remove as they were the imports required for the old style of `match`ing on the `Event` type.